banzai/flats.py

```diff
@@ -6,10 +6,26 @@
 from banzai.utils import stats
 from banzai.stages import Stage
 from banzai.calibrations import CalibrationStacker, CalibrationUser, CalibrationComparer
+import numpy as np
 
 logger = logging.getLogger('banzai')
 
 
+class FlatSNRChecker(Stage):
+    def __init__(self, runtime_context):
+        super(FlatSNRChecker, self).__init__(runtime_context)
+
+    def do_stage(self, image):
+        # Make sure the median signal-to-noise ratio is over 50 for the image other abort
+        flat_snr = np.median(image.data.signal_to_noise())
```

Comment:
I think we can actually do `image.primary_hdu.signal_to_noise()` - `image.data` just returns `image.primary_hdu.data`, which is a numpy array.

Maybe we can write a quick test for this one?